apps/docs/src/routes/docs/core/components/rating-group.mdx

```diff
@@ -0,0 +1,424 @@
+- **RatingGroup.Label**: The label that gives the user information on the rating group.
+- **RatingGroup.HiddenInput**: The native html input that is visually hidden in the rating group.
+- **RatingGroup.Description**: The description that gives the user more information on the rating group.
+- **RatingGroup.ErrorMessage**: The error message that gives the user information about how to fix a validation error on the rating group.
+
+The rating item consists of:
+
+- **RatingGroup.Item**: The root container for a rating item.
+- **RatingGroup.ItemControl**: The element that visually represents a rating item.
+- **RatingGroup.ItemLabel**: The label that gives the user information on the rating item.
+- **RatingGroup.ItemDescription**: The description that gives the user more information on the rating item.
+
+```tsx
+<RatingGroup>
+	<RatingGroup.Label />
+	<RatingGroup.Control>
+        <RatingGroup.Item>
+            <RatingGroup.ItemControl />
+            <RatingGroup.ItemLabel />
+            <RatingGroup.ItemDescription />
+        </RatingGroup.Item>
+    <RatingGroup.Control>
+	<RatingGroup.HiddenInput />
+	<RatingGroup.Description />
+	<RatingGroup.ErrorMessage />
+</RatingGroup>
+```
+
+## Example
+
+<Preview>
+	<BasicExample />
+</Preview>
+
+<TabsSnippets>
+  <TabsSnippets.List>
+    <TabsSnippets.Trigger value="index.tsx">index.tsx</TabsSnippets.Trigger>
+    <TabsSnippets.Trigger value="style.css">style.css</TabsSnippets.Trigger>
+  </TabsSnippets.List>
+  {/* <!-- prettier-ignore-start --> */}
+  <TabsSnippets.Content value="index.tsx">
+    ```tsx
+	import { RatingGroup } from "@kobalte/core/rating-group";
+    import "./style.css";
+
+    function App() {
+      return (
+        <RatingGroup class="rating-group">
+    		<RatingGroup.Label class="rating-group__label">Rate Us:</RatingGroup.Label>
+    		<RatingGroup.Control class="rating-group__control">
+    			<Index each={Array(5)}>
+    				{_ => (
+    					<RatingGroup.Item class="rating-group-item">
+    						<RatingGroup.ItemControl>
+    							<StarIcon />
+    						</RatingGroup.ItemControl>
+    					</RatingGroup.Item>
+    				)}
+    			</Index>
+    		</RatingGroup.Control>
+    	</RatingGroup>
+      );
+    }
+    ```
+
+  </TabsSnippets.Content>
+  <TabsSnippets.Content value="style.css">
+```css
+.rating-group {
+	display: flex;
+	flex-direction: column;
+	gap: 8px;
+}
+
+.rating-group\_\_label {
+color: hsl(240 6% 10%);
+font-size: 14px;
+font-weight: 500;
+user-select: none;
+}
+
+.rating-group\_\_description {
+color: hsl(240 5% 26%);
+font-size: 12px;
+user-select: none;
+}
+
+.rating-group\_\_error-message {
+color: hsl(0 72% 51%);
+font-size: 12px;
+user-select: none;
+}
+
+.rating-group\_\_control {
+display: flex;
+gap: 4px;
+}
+
+.rating-group-item {
+cursor: pointer;
+fill: hsl(240 6% 90%);
+transition: fill 200ms cubic-bezier(0.2, 0, 0, 1);
+}
+
+.rating-group-item[data-highlighted] {
+fill: hsl(200 98% 39%);
+}
+
+````
```

Comment:
nitpick 
there should be 3 backticks(`)

Reply:
Build fails with 3 backticks. Prettier keeps adding the 4th one.